yaksh/templates/yaksh/grade_user.html

```diff
@@ -156,22 +156,27 @@
     {% else %}
     <h5>Student answer: </h5>
     {% for answer in answers %}
-    {% if not answer.skipped %}
-    {% if answer.correct %}
+    {% if not answer.0.skipped %}
```

Comment:
It was already there, have removed it as I guess skipped answers should also be available for review